docs/topics/proc_updating-booster-deployment-configuration.adoc

```diff
@@ -0,0 +1,74 @@
+[id='proc_updating-booster-deployment-configuration_{context}']
+= Updating the Deployment Configuration of a Booster
+
+The deployment configuration for a booster contains information related to deploying and running the booster in OpenShift, such as route information or readiness probe location. 
+The deployment configuration of a booster is stored in a set of YAML files. 
+For boosters that use the Fabric8 Maven Plugin, the YAML files are located in the `src/main/fabric8/` directory. For boosters using Nodeshift, the YAML files are located in the `.nodeshift` directory.
+
+IMPORTANT: The deployment configuration files used by the Fabric8 Maven Plugin and Nodeshift do not have to be full OpenShift resource definitions. Both Fabric8 Maven Plugin and Nodeshift can take the provided files and infer some missing information to create a full OpenShift resource definition when targeting OpenShift.
```

Comment:
Is it OK in case of a call-out to put multiple sentences on a single line? :-)

No, this looks good to me. I think we should probably suggest to inspect the full definitions as generated by FMP/Nodeshift, or at least make a note where they can be found. In case of FMP, the generated resources are in `target/classes/META-INF/fabric8`.

Reply:
Actually one more: it's not 100% true that they can infer missing info just _when targeting OpenShift_. Again not sure about Nodeshift, but in case of FMP, it always generates both OpenShift and Kubernetes resources, and it adds missing info _always_ (even when targetting plain Kubernetes). I'd just remove the words "when targetting OpenShift".

Reply:
nodeshift just targets openshift

Reply:
Good point and I think removing the last 3 words ("when targetting OpenShift") won't change the meaning for Nodeshift.